### DISCUSSION TITLE
The web demo doesn't work

### DISCUSSION THREAD

**Discussion by ivan-developer-01** [lowlighter/metrics#1543] [Q&A] [answered]
I go to the https://metrics.lecoq.io/
It shows

![error](https://github.com/lowlighter/metrics/assets/127977316/e65794c5-ab1b-4688-bcb7-940f485d42a9)

Do you know how to fix it?

**Comment by Vault108** [comment]
@lowlighter  
It is currently hit or miss for me at the moment. I checked it a few times over the past 5 minutes:
The following happened: 
1) I got a 502 error
2) I got the same message you did
3) I got the same message you did, but had a search bar
4) I got it to work and it read Remaining GitHub requests: 4969 REST / 4947 GraphQL / 30 search

**Comment by lowlighter** [comment, marked as answer]
Hi ! 
This seems to be solved now (you may have to reload the cache too).
Apparently I misconfigured some network configuration and two of them had the same ip address. 
Sorry for the inconvenience 😅
